package.json

```diff
@@ -1,40 +1,31 @@
 {
-  "name": "liquibase",
-  "version": "0.0.3",
+  "name": "node-liquibase",
+  "version": "1.0.0",
```

Comment:
instead of making the jump to v1.0.0, what do you think about 0.1.0. This wasnt really a major change AFAIK, mostly just fixes by updating internal dependencies, right?

Reply:
more info on semantic versioning:
https://semver.org/